BD-4437: Clarify that seed emails don't impact frequency capping (#9373)
braze-inc/braze-docs: _docs/_user_guide/administrative/app_settings/internal_groups_tab.md

{% alert note %}
[SMS]({{site.baseurl}}/user_guide/message_building_by_channel/sms/) test messages can only be sent to valid phone numbers in the database.
{% endalert %}

You can select individual Braze users or as many internal groups to send the message to. If your message includes any Liquid or other dynamic personalization, Braze will use the attributes available for each user to personalize the message contents. For users who have no attributes, Braze will use the default value set.

Additionally, if you preview the message as a random user, custom user, or existing user, you can send that previewed version instead. Clearing the checkbox allows you to send based on each user's attributes versus the previewed version.

If you use an IP pool to send out an email, you can select which IP pool you would like the email to be sent from by selecting the pool from the dropdown available.

![The Test section of the in-app message editor to select the Content Test Group.]({% image_buster /assets/img_archive/content_test_preview.png %}){: style="max-width:60%" }

### Seed Groups

> Seed Groups supported for the email channel only and allow you to send a copy of each email variant message to members of that group. 

Reviewer: ```suggestion
Seed Groups are only supported for the email channel. You can add users to a Seed Group to send copies of each email variant message to all members of the group.
```

Reviewer: fixed typo + separated the sentence.

Seed Groups aren't available for API campaigns, but you can include Seed Groups using an API-triggered entry in the campaign. You can use this to measure deliverability metrics and to keep a record of your email content for historical and archival purposes. 

After creating an internal group and tagging it to be used as a Seed Group, you can select it from the **Target Audiences** step of the campaign editor, or on the **Send Settings** step in a Canvas. 

Seed emails will have `[SEED]` appended to the start of the email subject line. Note that seed emails **do not**:

- Increment sends in dashboard analytics.
- Impact email analytics or retargeting. 
- Update a user profile's **Campaign Received** list.
- Impact frequency capping.

{% alert tip %}
If your Seed Group members report not seeing the message in their inbox, check that they're listed in the internal group, verify that your subject lines are different and that Gmail has not bundled the emails together, or have them check their spam folders.
{% endalert %}

> #### For campaigns

Reviewer: Might be worth putting these two options in tabs 👍🏽
